1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,74 @@
-function setAlarm() {}
+const display = document.querySelector("#timeRemaining");
+var audio = new Audio("alarmsound.mp3");
+audio.loop = true;
```

Comment:
Think this needs to be removed - the spec only calls for the audio to be played once